src/DistanceCalculatorAVX.cpp

```diff
@@ -377,45 +383,57 @@ double DistanceCalculator::newCalcDNA(int a, int b) {
             // TODO: does this increase from +=4 to +=8? TEST
         }
 
-        /*gather transversion counts*/
+        /*gather transversion counts:
+        Compute the absolute differences of packed unsigned 8-bit
+        integers in a and b, then horizontally sum each
+        consecutive 8 differences to produce four unsigned 16-bit
+        integers, and pack these unsigned 16-bit integers in the
+        low 16 bits of 64-bit elements in dst.
+        **in avx this gives 4 16-bit integers, not 2*/
```

Comment:
This comment is a little confusing. We're packing four 16-bit integers into a single 64-bit element? Or one per 64-bit element?

Reply:
It's packing 1 integer into the low 16 bits of each of the 64 bit blocks... Slightly confusing phrasing courtesy of the Intel Intrinsics guide.